ui/raidboss/data/05-shb/raid/e8s.js

```diff
@@ -75,11 +77,23 @@
           return {
             en: 'Biting Frost Next',
             fr: 'Taillade de givre bientot',
+            ko: '다음: Biting/スラッシュ',
           };
         }
         return {
           en: 'Driving Frost Next',
           fr: 'Percée de givre bientot',
+          ko: '다음: Driving/スラスト',
+        };
+      },
+      tts: function(data) {
+        if (data.firstFrost == 'biting') {
+          return {
+            ko: '다음: 바이팅 스라슈',
+          };
+        }
+        return {
+          ko: '다음: 드라이빙 스라스토',
```

Comment:
This part exists because of Korean TTS does not read Japanese.
There are English and Japanese in Korean alertText because Korean does not yet have a translated skill name and Korean who plays in Global(SE) Server, some are using English and others, Japanese.

Reply:
TTS for one language will remove TTS for all languages not listed

Reply:
ah, really? I think that should be fixed. exceptional TTS is not always same time needed for all languages.

Reply:
Yea; I believe it's a bug: https://github.com/quisquous/cactbot/issues/1076

Reply:
Yeah, this is the way that tts has always worked.  Probably the logic about when to use the tts or sounds needs to have an eye on it to meet people's expectations better.  For now, I would not add a separate tts section.